src/MooLite/core/character/Character.ts

```diff
@@ -0,0 +1,13 @@
+export interface Character {
```

Comment:
I would prefer for `Character` to be a class so we can add helper methods to it if needed.

It could implement an interface with this content called `CharacterDetail` to ensure it has the right attributes

That shouldn't affect code relying on `game.character`